Library/Homebrew/tap.rb

```diff
@@ -876,6 +863,31 @@ def tap_migrations
     end
   end
 
+  sig { returns(T::Hash[String, T::Array[String]]) }
+  def reverse_tap_migrations_renames
+    @reverse_tap_migrations_renames ||= tap_migrations.each_with_object({}) do |(old_name, new_name), hash|
```

Comment:
Needs to be cleared in `#clear_cache`.

Reply:
Good catch. I added it to the https://github.com/Homebrew/brew/pull/16863/commits/d4a273443c0d5bb7d84d4c96a5118aefefac6599 commit.